js-green-licenses.json

```diff
@@ -0,0 +1,5 @@
+{
+  "packageWhitelist": [
+    "log-driver"
```

Comment:
You mentioned that Modelo is also a problem.  Why isn't it listed here?

Reply:
Modelo was resolved over the weekend: https://github.com/kevinconway/Modelo.js/pull/20